packages/kbn-optimizer/limits.yml

```diff
@@ -105,6 +105,7 @@ pageLoadAssetSize:
   indexPatternFieldEditor: 90489
   osquery: 107090
   fileUpload: 25664
+  fileDataVisualizer: 27530
```

Comment:
Lots of new code, 27kb isn't a ton and I assume that y'all are async importing everything based on the `lazy_load_bundle` code, so LGTM